src/components/policy/policy_external/test/update_status_manager_test.cc

```diff
@@ -72,18 +73,53 @@ class UpdateStatusManagerTest : public ::testing::Test {
   void TearDown() OVERRIDE {}
 };
 
-// TODO(AKutsan) : APPLINK-31222 use miliseconds as parameter of OnUpdateSentOut
-// function
-TEST_F(
-    UpdateStatusManagerTest,
-    DISABLED_OnUpdateSentOut_WaitForTimeoutExpired_ExpectStatusUpdateNeeded) {
+namespace {
+class WaitAsync {
+ public:
+  WaitAsync(const uint32_t count, const uint32_t timeout)
+      : count_(count), timeout_(timeout), cond_var_() {}
+
+  void Notify() {
+    count_--;
+    cond_var_.NotifyOne();
+  }
+
+  void Wait(sync_primitives::AutoLock& auto_lock) {
```

Comment:
@LitvinenkoIra I propose make this method boolean. return  True in case if all shots are called correctly, And return false in case of timeout